api/views/CloudRecordSummaryView.py

```diff
@@ -49,6 +53,44 @@ def get(self, request, format=None):
         """
         logger = logging.getLogger(__name__)
 
+        try:
+            client_token = self._request_to_token(request)
+        except KeyError:
+            logger.error("No AUTHORIZATION header provided, authentication failed.")
+            return Response(status=401)
+        except IndexError:
+            logger.error("AUTHORIZATION header provided, but not of expected form.")
+            logger.error(request.META['HTTP_AUTHORIZATION'])
+            return Response(status=401)
+
+        logger.info("%s Authenticated", client_token)
+
+        try:
+            client_id = self._token_to_id(client_token)
+        except urllib2.HTTPError, e:
+            logger.error('HTTPError = ' + str(e.code))
+            logger.error('Could not Authorize.')
+            return Response(status=401)
+        except urllib2.URLError, e:
+            logger.error('URLError = ' + str(e.reason))
+            logger.error('Could not Authorize.')
+            return Response(status=401)
+        except httplib.HTTPException:
+            logger.error('HTTPException')
+            logger.error('Could not Authorize.')
+            return Response(status=401)
+        except KeyError:
+            logger.error("No client id in IAM response, likely token has expired")
+            return Response(status=401)        
+
+        logger.info("Token identifed as %s", client_id)
```

Comment:
You might want to change this to `DEBUG` level as you're already logging the `client_id`, whether they are authorized or not, in the next few lines.